src/shared/clients/devenvClient.ts

```diff
@@ -103,6 +108,67 @@ export class DevEnvClient implements vscode.Disposable {
         // `Authorization` _should_ have two parameters (RFC 7235), MDE should probably fix that
         headers: { Authorization: this.authToken },
     })
+
+    #lastKnownActivityTimestamp: number | undefined
```

Comment:
Can we avoid dealing with undefined for this variable:

```suggestion
    #lastKnownActivityTimestamp: number | 0
```

Reply:
0 implies the last user activity was at 0, and undefined implies no user activity was set yet. This distinction is important.